style.css

```diff
@@ -93,3 +93,119 @@ main {
   width: 100%;
   background-color: black;
 }
+
+.container {
+  display: grid;
+  place-items: center;
+  padding: 1rem;
+}
+
+h1 {
+  margin-bottom: 25px;
+  line-height: 1.2rem;
+  color: wheat;
+}
+
+.slider {
+  display: block;
+  /* margin: auto; */
+  position: relative;
+  /* display: grid; */
+  /* place-items: center; */
+  max-width: 70%;
+  height: 600px;
+  overflow: hidden;
+}
+
+.slide {
+  display: block;
+  /* display: grid; */
+  /* place-items: center; */
+  height: 100%;
+  user-select: none;
+  display: flex;
+  height: 600px;
+}
+
+.slide img {
+  background: rgb(172, 172, 220);
+  flex: 1 0 100%;
+  object-fit: cover;
+}
+
+.slider__btn-container {
+  z-index: 10000000;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  /* display: none; */
+}
+
+.slider__btn {
+  border: 10px solid red;
+  border: none;
+  height: 60px;
+  width: 60px;
+  text-align: center;
+  position: absolute;
+  z-index: 1000;
+  top: 48%;
```

Comment:
Instead of top: 48% try:
`top: 50%;
transform: translateY(-50%)`
This would make it perfectly centered.